src/main/java/frc/robot/subsystems/Crane.java

```diff
@@ -307,6 +310,19 @@ private void resetCrane() {
     );
   }
 
+  private void movePivotWithAbsEncoder(double pivotAngle) {
+    Translation2d position = getPositionWithAbsEncoder();
+    Translation2d deviation = getDeviation(position);
+    
+    m_aController.setGoal(pivotAngle);
+
+    scaleAHConstraints(position, deviation);
+    double aVelocity = m_aController.calculate(position.getX());
+    SmartDashboard.putNumber("Pivot Goal", Units.radiansToDegrees(m_aController.getGoal().position));
+    m_pivotPID.setReference(aVelocity, ControlType.kMAXMotionVelocityControl,
+      CraneConstants.kPivotMotorVelocityPIDFSlot.slot());
+  }
+  
```

Comment:
We can initialize the pivot motor encoder during `Crane` construction to equal the value reported by the absolute encoder, then never use the absolute encoder again. That provides two benefits:

1) Crane homing logic is simpler because we immediately know the pivot position.
2) We don't have to hit a limit to home the pivot.

It looks like you ripped out voltage-based pivot control and then wrapped position-based control around velocity-based control. But we really want to *replace* voltage-based control with position-based control.

Note that this PR left some of the voltage-based pivot PIDF in place. That should be refactored as position-based PIDF configuration.